releases/latest/full/Dockerfile.ubuntu.adoptopenjdk8

```diff
@@ -16,6 +16,7 @@ LABEL org.opencontainers.image.authors="Arthur De Magalhaes, Chris Potter" \
 
 COPY helpers /opt/ol/helpers
 COPY fixes/ /opt/ol/fixes/
+COPY licenses /licenses
```

Comment:
@arthurdm while adding the Java 11 support to ubuntu images, noticed that the license directory was not being copied for ubuntu images. I think it should be copied. Please let me know if it shouldn't be copied.

Reply:
yup, good catch.  In WL we switched to downloading the Licenses for Ubuntu instead of doing a `COPY`, but that was because they changed with every release. I believe the OL license file is always the same, so that shouldn't trigger `diff`s when pushing into Docker Hub.